### PR TITLE
POC-946: Changed collections limits to be in ETH

### PR DESCRIPTION
**Pull request by DioPires**
## Purpose of this PR 🎯

-   [x] Feature;
-   [ ] Bugfix;
-   [ ] Tests;
-   [ ] Refactoring;
-   [ ] Build or CI/CD; 
-   [ ] Documentation;
-   [ ] Code Styling;
-   [ ] Other. Please describe:

## Changes 📝

* Change the limit of the collections to be expressed in ETH instead of being a percentage of borrowed liquidity
* The limits can be seen [here](https://zharta.atlassian.net/wiki/spaces/PROD/pages/495845377/Liquidity+Origination+Controls+v.2)

### CODE REVIEW

**Review by cfcfs — APPROVED** [review]
Shouldn't the full deployment scripts be updated with the new constructor for `LiquidityControls`?
Other than that, LGTM!

**Review by cfcfs — APPROVED** [review]
LGTM 👍

File: scripts/deployment_liquidations_peripheral.py [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+def load_contracts(env: str) -> dict:
+    config_file = f"{Path.cwd()}/configs/{env}/contracts.json"
+    with open(config_file, "r") as f:
+        contracts = json.load(f)["tokens"]["WETH"]
+
+    return {key: NAME_TO_CONTRACT[key].at(contracts[key]["contract"]) for key in contracts}
+
+
+def dev():
```
Comment (ametade) [inline scripts/deployment_liquidations_peripheral.py:41]: the dev(), int() and prod() functions seem to do the same thing. Would it make sense to create a new functions that gets as argument the stage and does what these three functions are doing?

File: scripts/deployment_liquidity_controls.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        "wow_instance": ERC721.at(contracts[4]["contract"]),
+        "mutant_instance": ERC721.at(contracts[5]["contract"]),
+        "veefriends_instance": ERC721.at(contracts[6]["contract"]),
+        "pudgypenguins_instance": ERC721.at(contracts[7]["contract"]),
+        "bayc_instance": ERC721.at(contracts[8]["contract"]),
+        "wpunks_instance": ERC721.at(contracts[9]["contract"])
+    }
+
+
+def dev():
```
Comment (ametade) [inline scripts/deployment_liquidity_controls.py:62]: See my previous comment about dev(), int() and prod() being replaced by one function.

File: scripts/deployment_liquidations_peripheral.py (hunk 1 above)
Comment (DioPires) [reply scripts/deployment_liquidations_peripheral.py:41]: They do seem to do the same thing for the most part now but depending on the environment, the configuration values of some of the contracts may differ, as they have in the past and may well again for testing purposes in the future. In addition, `prod()` usually has a couple more transactions that aren't needed in `dev()` and `int()`. We can simplify this but at the cost of having to revert that simplification if configuration values change in the future again.

For example, in this specific contract, we may have grace period and lenders period durations that are different for mainnet.

What do you think? @ametade @cfcfs

**Pull request merged**